applications/FemToDemApplication/custom_processes/compute_normalized_free_energy_on_nodes_process.hpp

```diff
@@ -165,12 +165,13 @@ class ComputeNormalizedFreeEnergyOnNodesProcess : public Process
 
     	Vector lengths = ZeroVector(6);
         for (unsigned int edge = 0; edge < 6; edge++) { // Loop over edges
-            const double X1 = rGeometry.Edges()[edge][0].X0();
-            const double X2 = rGeometry.Edges()[edge][1].X0();
-            const double Y1 = rGeometry.Edges()[edge][0].Y0();
-            const double Y2 = rGeometry.Edges()[edge][1].Y0();
-            const double Z1 = rGeometry.Edges()[edge][0].Z0();
-            const double Z2 = rGeometry.Edges()[edge][1].Z0();
+            auto& edges = rGeometry.Edges();
```

Comment:
r_edge, and declare it out of the loop!!!

Reply:
yeah, sorry